### PR TITLE
[plugin.video.southpark_unofficial] v0.4.8

### PR DESCRIPTION
**Pull request by wargio**
### Description

Fix busy dialog during playback
- pull request #9 from anxdpanic/busydialogfix

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by wargio** [comment]
i don't know why it had exec permissions on those files.

File: plugin.video.southpark_unofficial/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.southpark_unofficial" name="South Park" version="0.4.7" provider-name="Deroad">
+<addon id="plugin.video.southpark_unofficial" name="South Park" version="0.4.8" provider-name="Deroad">
     <requires>
         <import addon="xbmc.python" version="2.1.0"/>
```
Comment (Rechi) [inline plugin.video.southpark_unofficial/addon.xml]: > `WARN: For helix it is advised to set xbmc.python version to 2.19.0`
```suggestion
         <import addon="xbmc.python" version="2.19.0"/>
```

**Comment by Rechi** [comment]
@wargio please update the following numliteral syntax change. Also the `website` tag in addon.xml should point to the content site, not to your personal site.
```
--- a/plugin.video.southpark_unofficial/default.py
+++ b/plugin.video.southpark_unofficial/default.py
@@ -41,7 +41,7 @@
 viewMode = str("504")
 addonpath = xbmc.translatePath('special://temp/temp')
 if not os.path.isdir(addonpath):
-	os.mkdir(addonpath, 0755)
+	os.mkdir(addonpath, 0o755)
 
 
 def index():
```

**Comment by wargio** [comment]
it's the first time that i heard that about the website. i'll patch it!

**Comment by Rechi** [comment]
`plugin.video.southpark_unofficial/default.py` has inconsistent use of tabs and spaces in indentation, please fix that in the next update.

**Pull request merged**